lesson-000-web-server/README.md

```diff
@@ -45,3 +79,17 @@ And another one
 $ curl http://localhost:8080/hello/again/
 Greeting from alejandrox1-machine1!
 ```
+
+### Next steps (optional)
+
+As shown, we currently have one (root) route, which means: it works but not very practical.
+
+We would like to extend the current lesson with:
```

Comment:
We should reword this. Something like "Couple interesting things we can do".
This is more of an "excessive for the reader" so up to them.